### PR TITLE
DE38899 Lang terms for 2005

### PR DESCRIPTION
**Pull request by emil-furniss**
Also includes the lang term updates for 03 and 04

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-assignment-editor/lang/da-dk.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* eslint quotes: 0 */
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/lang/da-dk.js:1]: This file seems half translated

Comment (emil-furniss) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/lang/da-dk.js:1]: Yeah, I'm sure it'll come in later. Danish is always a weird one.

File: components/d2l-activity-editor/d2l-activity-rubrics/lang/ko.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default {
+	"btnAddRubric": "루브릭 추가", //text for add rubric button
+	"btnCreateNew": "새로 만들기", //Text for create new dropdown
+	"btnAddExisting": "기존 항목 추가", //Text for Add Existing dropdown
+	"hdrRubrics": "루브릭", //Header for the rubrics section
+	"btnAttachRubric": "루브릭 부착", //Button for the attach new rubric overlay
+	"btnCancel": "취소", //Button for canceling out of the attach new rubric overlay
+	"txtAddExisting": "기존 항목 추가", //Title for the attach rubrics dialog,
+	"txtNoRubricAdded": "추가된 루브릭 없음", // rubric summary for no rubrics
+	"txtRubricsAdded": "{수, 복수, =1 {루브릭 1개 추가} 기타 {{count}개 루브릭 추가}}", // count of asoociated rubrics
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-rubrics/lang/ko.js:12]: wrongly translated the 1/other :(

Comment (emil-furniss) [reply components/d2l-activity-editor/d2l-activity-rubrics/lang/ko.js:12]: good catch!

File: components/d2l-activity-editor/lang/ko.js [hunk 3]
```diff
@@ -2,32 +2,54 @@
 
 export default {
 	"btnEditReleaseConditions": "릴리스 조건 편집", // edit release conditions button
+	"btnAddReleaseCondition": "릴리스 조건 추가", // add release condition button
+	"btnCreateNew": "새로 만들기", // create new button
+	"btnAddExisting": "기존 항목 추가", // add existing button
+	"btnRemoveCondition": "조건 제거", // remove condition button
+	"lblConditionsOperator": "이 항목을 보려면 사용자가 다음을 충족해야 합니다.", // conditions operator label
+	"txtNumReleaseConditions": "{수, 복수, =1 {{count}개 릴리스 조건} 기타 {{count}개 릴리스 조건}}", // num release condition text
```
Comment (wongvincent) [inline components/d2l-activity-editor/lang/ko.js:10]: translated wrong :(

File: components/d2l-activity-list-item/lang/da-dk.json [hunk 4]
```diff
@@ -1,3 +1,4 @@
 {
+   "clickToViewActivity" : "Click to view activity",
```
Comment (wongvincent) [inline components/d2l-activity-list-item/lang/da-dk.json:2]: missed translation?

File: components/d2l-quick-eval/lang-experimental/da-dk.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+export const LangEn = {
```
Comment (wongvincent) [inline components/d2l-quick-eval/lang-experimental/da-dk.js:1]: oh, I'm guessing da-dk is not supported yet, so it's not translated?

Comment (emil-furniss) [reply components/d2l-quick-eval/lang-experimental/da-dk.js:1]: something like that

File: components/d2l-quick-eval/lang/da-dk.json [hunk 6]
```diff
@@ -7,20 +7,22 @@
    "cancel" : "Cancel",
    "caughtUp" : "Du er helt med!",
    "checkBackOften" : "Kom tilbage med jævne mellemrum for at se nye afleveringer.",
-   "checkBackOftenCourseLevel": "Check the {startTag}Multi-Course Quick Eval{endTag} to see if there are new submissions in your other courses.",
+   "checkBackOftenCourseLevel" : "Check the {startTag}Multi-Course Quick Eval{endTag} to see if there are new submissions in your other courses.",
```
Comment (wongvincent) [inline components/d2l-quick-eval/lang/da-dk.json:10]: same comment. da-dk file missing translations. expected i guess

File: components/d2l-activity-list-item/lang/da-dk.json (hunk 4 above)
Comment (wongvincent) [reply components/d2l-activity-list-item/lang/da-dk.json:2]: nvm it's da-dk

**Pull request merged**